### PR TITLE
Fixes/cleanup to manifest and config

### PR DESCRIPTION
**Pull request by VinceMalone**
1. File paths in manifest.webapp were not inheriting the value from the `path` member in config. This is because, in [helpers.js](https://github.com/haydenbleasel/favicons/blob/7f099d574170d55fd76161dcc5c8ad66e9ac17bd/helpers.js#L190), `_.map()` creates a new array and does not mutate the passed in array (among other reasons).
2. The `density` member in manifest.json has been changed to a number. While technically having the value as a string *should* work, the [specification](https://www.w3.org/TR/appmanifest/#density-member) alludes to it being a number type.
3. Added a `<link>` element for the coast icon.
4. #69 #89

### CODE REVIEW

**Comment by jbmoelker** [comment]
+1 Google Chrome team also updated `density` member their `manifest.json` sample accordingly: https://github.com/GoogleChrome/samples/pull/98

File: helpers-es5.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-function _typeof(obj) { return obj && typeof Symbol !== "undefined" && obj.constructor === Symbol ? "symbol" : typeof obj; }
+var _typeof = typeof Symbol === "function" && typeof Symbol.iterator === "symbol" ? function (obj) { return typeof obj; } : function (obj) { return obj && typeof Symbol === "function" && obj.constructor === Symbol ? "symbol" : typeof obj; };
```
Comment (haydenbleasel) [inline helpers-es5.js]: What happened here? Part of a Babel update?

File: helpers-es5.js [hunk 2]
```diff
@@ -187,7 +188,6 @@ var path = require('path'),
                     print('Files:create', 'Creating file: ' + name);
                     if (name === 'manifest.json') {
                         properties.name = options.appName;
-                        properties.short_name = options.appName;
```
Comment (haydenbleasel) [inline helpers-es5.js]: How come `properties.short_name` is being deleted?

**Comment by haydenbleasel** [comment]
Awesome, loving these commits mate :) Just gotta sort out a few questions then I'm good to merge

**Comment by haydenbleasel** [comment]
Also @phbernard You should cross reference this with your code :)

File: helpers-es5.js (hunk 1 above)
Comment (VinceMalone) [inline helpers-es5.js]: Yea; babel.

File: helpers-es5.js (hunk 2 above)
Comment (VinceMalone) [inline helpers-es5.js]: This file was compiled (via babel) from helpers.js and `properties.short_name` doesn't seem to exist there.

Comment (haydenbleasel) [inline helpers-es5.js]: Ahh I see. I think someone PR'ed `properties.short_name = options.appName;` into `helpers-es5` but not `helpers.js`. Do you reckon you could add it to the ES6 version then re-compile? Thanks mate :)

**Pull request merged**